docs/pages/inboxes/build-inbox.md

```diff
@@ -0,0 +1,172 @@
+## Build the inbox
+
+:::warning[🚧 XMTP V3 Alpha]
+
+The XMTP V3 SDKs are in **alpha** status and ready for you to start experimenting with.
```

Comment:
"Experiment" feels off -- the reader is ready to build, even if v3 isn't ready for them.